src/Microsoft.AspNetCore.ResponseCaching/Internal/DistributedResponseCacheStore.cs

```diff
@@ -42,17 +60,51 @@ public DistributedResponseCacheStore(IDistributedCache cache)
             catch { }
```

Comment:
We will probably log exceptions, such as when the connection is unreliable. Adding logging is a todo item https://github.com/aspnet/ResponseCaching/issues/42